lib/install/management/delete.go

```diff
@@ -120,14 +120,14 @@ func (d *Dispatcher) getComputeResource(vmm *vm.VirtualMachine, conf *config.Vir
 	return rp, nil
 }
 
-func (d *Dispatcher) getImageDatastore(vmm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec) (*object.Datastore, error) {
+func (d *Dispatcher) getImageDatastore(vmm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec, force bool) (*object.Datastore, error) {
 	var err error
-	if len(conf.ImageStores) == 0 {
-		if !d.force {
+	if conf == nil || len(conf.ImageStores) == 0 {
+		if !force {
 			err = errors.Errorf("Cannot find image stores from configuration")
 			return nil, err
 		}
-		log.Warnf("Cannot find image stores from configuration, attempting to delete from vm datastore")
+		log.Debugf("Cannot find image stores from configuration, attempting to find from vm datastore")
```

Comment:
This is really minor, and I'm pretty sure I wrote this log message originally so this is actually my error, but can we change that comma to a semicolon please? It's grammatically incorrect as it is and it's bugging me ;D